src/styles/mixins/layout.scss

```diff
@@ -11,3 +11,16 @@
     padding-right: 4rem;
   }
 }
+
+@mixin fullwidth-messages {
```

Comment:
I've attempted to be somewhat thoughtful here, building on existing responsive rules for the width of elements. The idea here is to constrain the width of the toast messages to the max-width of the overall container (without this constraint, toast messages are the full width of the screen) and center the message on the page.